plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -387,15 +418,30 @@ class FeatureDevController(
         chatSessionStorage.deleteSession(tabId)
 
         newTabOpened(tabId)
-        if (isException != null && !isException) {
-            messenger.sendAnswer(
-                tabId = tabId,
-                messageType = FeatureDevMessageType.Answer,
-                message = message("amazonqFeatureDev.chat_message.ask_for_new_task")
-            )
+
+        if (prefilledPrompt != null && isException != null && !isException) {
```

Comment:
what handles the exception case?

Reply:
I'm not sure. This existed before, I was just following the previous implementation/pattern and it doesn't seem like there is any exception handling done. I can remove if you want